api/validate.go

```diff
@@ -14,6 +15,12 @@ func (s *Server) validateBalances(ctx context.Context) {
 	if s.Offline {
 		return
 	}
+	switch os.Getenv("DISABLE_BALANCE_VALIDATION") {
+	case "false", "off", "0", "":
```

Comment:
what is "" case?

Reply:
When the `DISABLE_BALANCE_VALIDATION` environment variable hasn't been set, `os.Getenv` will return an empty string, i.e. `""`